packages/documentation/src/content/docs/concepts/open-payments/grant-interaction.mdx

```diff
@@ -2,11 +2,32 @@
 title: Grant Interaction Flow
 ---
 
-import LargeImg from '/src/components/docs/LargeImg'
+import MermaidWrapper from '/src/components/docs/MermaidWrapper'
 
 ## Sequence Diagram
 
-<LargeImg src='/img/grant-interaction-flow.svg' alt='Sequence diagram' />
+<MermaidWrapper client:load>
+```mermaid
```

Comment:
The arrows are not displayed for some reason.

Reply:
> The arrows are not displayed for some reason.

let me check this.